_authors/dchanp.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Darshit Chanpura
+short_name: dchanp
+photo: '/assets/media/authors/dchanp.jpg'
+github: DarshitChanpura
+---
+
+**Darshit Chanpura** is an AWS Software Development Engineer and a maintainer of the OpenSearch Security and Security-Dashboards plugin.
```

Comment:
The style job fails with:
```
  Error: [vale] reported by reviewdog 🐶
  [OpenSearch.Spelling] Error: Darshit. If you are referencing a setting, variable, format, function, or repository, surround it with tic marks.
  
  Raw Output:
  {"message": "[OpenSearch.Spelling] Error: Darshit. If you are referencing a setting, variable, format, function, or repository, surround it with tic marks.", "location": {"path": "_authors/dchanp.markdown", "range": {"start": {"line": 8, "column": 3}}}, "severity": "ERROR"}
  Error: [vale] reported by reviewdog 🐶
  [OpenSearch.Spelling] Error: Chanpura. If you are referencing a setting, variable, format, function, or repository, surround it with tic marks.
  
  Raw Output:
  {"message": "[OpenSearch.Spelling] Error: Chanpura. If you are referencing a setting, variable, format, function, or repository, surround it with tic marks.", "location": {"path": "_authors/dchanp.markdown", "range": {"start": {"line": 8, "column": 11}}}, "severity": "ERROR"}
  reviewdog: found at least one result in diff
Error: reviewdog exited with status code: 1
```

Seems like it an issue with `*` around Darshit Chanpura to mark it as bold. I've followed suit from markdowns of other authors. Am I missing something here?